src/evict/evict_lru.c

```diff
@@ -2315,9 +2316,11 @@ __wt_cache_eviction_worker(WT_SESSION_IMPL *session, bool busy, bool readonly, d
     for (initial_progress = cache->eviction_progress;; ret = 0) {
         /*
          * If eviction is stuck, check if this thread is likely causing problems and should be
-         * rolled back. Ignore if in recovery, those transactions can't be rolled back.
+         * rolled back. Ignore if in recovery or verifying metadata, those transactions can't be
+         * rolled back.
```

Comment:
The way this is phrased, "verifying metadata" is somehow transactional operations which isn't really the case, I think.